packages/admin/cms-admin/src/generator/future/utils/generateFieldList.ts

```diff
@@ -0,0 +1,69 @@
+import { IntrospectionField, IntrospectionQuery, IntrospectionType } from "graphql";
+import objectPath from "object-path";
+
+type FieldsObjectType = { [key: string]: FieldsObjectType | boolean | string };
+const recursiveStringify = (obj: FieldsObjectType, objectSuffix?: string): string => {
```

Comment:
`objectSuffix` isn't used, right? Can we remove it? Also, this function seems similar to `convertObjectToStructuredString`, could we use the same function for both use cases?

Reply:
yes, that's right. `objectSuffix` can be removed.

regarding reusing the same function: there are two types of format

```
product {
    packageDimension {
        height
        width
    }
}
```

and 

```
product: {
    packageDimension: {
        height: string;
        width: string;
    }
}
```

initially I tried to solve both cases with the same function using objectSuffix. But then I reminded myself to keep it simple and differentiate between use-cases: prefer copy over complicated. And `recursiveStringify` is only needed for graphql formatting, and `convertObjectToStructuredString` is used in different places... so there are now two functions for two use-cases, with similar code.